intelmq/bots/parsers/shadowserver/config.py

```diff
@@ -1280,3 +1281,28 @@ def validate_fqdn(value):
         'classification.identifier': 'opencwmp',
     }
 }
+
+accessible_vnc = {
+    'required_fields': [
+        ('time.source', 'timestamp', add_UTC_to_timestamp),
+        ('source.ip', 'ip'),
+        ('source.port', 'port'),
+    ],
+    'optional_fields': [
+        ('source.reverse_dns', 'hostname'),
+        ('source.asn', 'asn'),
+        ('source.geolocation.cc', 'geo'),
+        ('source.geolocation.region', 'region'),
+        ('source.geolocation.city', 'city'),
+        ('extra.', 'naics', invalidate_zero),
+        ('extra.', 'sic', invalidate_zero),
+        ('extra.', 'banner', validate_to_none),
+        ('extra.', 'product', validate_to_none),
+    ],
+    'constant_fields': {
+        'protocol.transport': 'tcp',
+        'protocol.application': 'vnc',
+        'classification.type': 'vulnerable service',
```

Comment:
actually this is a problem with all "openly accessible services":
  * for some we know that this is not a good idea / best practice and it should be not allowed (e.g. open recursive DNS) - since you can argue that it harms others.
  * some others are open  but "meh" not so pretty. But it's hard to argue that they are vulnerable. Do we know it? This might be the very latest and most current patched VNC server with password and a good crypto authentication library. We don't know. 

In short: we might have come up with a new category . Let's call it "accessible services" or  "potentially  vulnerable services" . This would also require new texts.

-> I think I want to discuss this with the team and community.